sympy/sets/handlers/issubset.py

```diff
@@ -133,3 +134,24 @@ def is_subset_sets(a, b): # noqa:F811
 @dispatch(Rationals, Range)
 def is_subset_sets(a, b): # noqa:F811
     return False
+
+@dispatch(ConditionSet, ConditionSet)
+def is_subset_sets(a, b):  # noqa:F811
+    if a.condition == b.condition and a.base_set.is_subset(b.base_set):
+        return True
```

Comment:
This would work better as `return a.base_set.is_subset(b.base_set)` because then it can return True, False or None as needed and there would be no need to to evaluate the code below. You should test examples where the relationship between the base sets give True, False and None.

It's not enough to check just that the conditions are equal though. The symbols would also need to be equal:
```julia
In [7]: x, y, z = symbols('x, y, z', real=True)                                                                                                

In [8]: S1 = ConditionSet(x, x<y, Reals)                                                                                                       

In [9]: S2 = ConditionSet(y, x<y, Reals)                                                                                                       

In [10]: S1.as_relational(z)                                                                                                                   
Out[10]: z < y

In [11]: S2.as_relational(z)                                                                                                                   
Out[11]: x < z
```
With this PR:
```julia
In [12]: S1.is_subset(S2)                                                                                                                      
Out[12]: True
```
Can you add tests for that?

Reply:
Also
```julia
In [1]: x, y, z = symbols('x, y, z', real=True)                                                                                                

In [2]: S1 = ConditionSet(x, x<y, Reals)                                                                                                       

In [3]: S1.rewrite(Intersection)                                                                                                               
---------------------------------------------------------------------------
NotImplementedError 
```

Reply:
That's because of:
```python
>>> from sympy.abc import x, y
>>> (x < y).as_set()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/enojb/current/sympy/sympy/sympy/logic/boolalg.py", line 161, in as_set
    raise NotImplementedError("Sorry, as_set has not yet been"
NotImplementedError: Sorry, as_set has not yet been implemented for multivariate expressions
```